html/en-US/privacypolicy.html

```diff
@@ -19,10 +19,11 @@
     <li class="terms-title">2. INFORMATION COLLECTED AND METHOD OF OBTAINING:</li>
     <p>For use of the TEKSTO application services and for restricted application access, the following information may be requested:</p>
     <ul>
-      <li>2.1. "Personal Information": for subscription and / or registration for full enjoyment of the tools offered by TEKSTO, users must provide certain Personal Information necessary for their identification, such as contact information, age, location, spoken language and gender, email, and so on. For the acquisition of services, financial information will be requested. This information is provided voluntarily and its truthfulness, timeliness and completeness is the sole responsibility of the users.</li>
+      <li>2.1. "Personal Information": for subscription and / or registration for full enjoyment of the tools offered by TEKSTO, users must provide certain Personal Information necessary for their identification, such as contact information, age, location, spoken language and gender, email, profile picture. Some information may be collected from third part applications required to make an easier login process, for example, name, email and profile picture collected from sign up with Facebook. For the acquisition of services, financial information will be requested. This information is provided voluntarily and its truthfulness, timeliness and completeness is the sole responsibility of the users. The user can require for a deletion of personal data from our database via <a href="suporte@teksto.com">Contact Us</a>, however, some information is essential for TEKSTO's service to work properly, for example email, and removing this kind of data may make it impossible for the user to keep using the TEKSTO app. In case it occurs, the user will be forwarded to the complete deletion of their account from our system.</li>

```

Comment:
```suggestion
      <li>2.1. "Personal Information": for subscription and/or registration for full enjoyment of the tools offered by TEKSTO, users must provide certain Personal Information necessary for their identification, such as contact information, age, location, spoken language, and gender, email, profile picture. Some information may be collected from third-party applications required to make an easier login process, for example, name, email, and profile picture collected from the sign up with Facebook. For the acquisition of services, financial information will be requested. This information is provided voluntarily and its truthfulness, timeliness, and completeness are the sole responsibility of the users. The user can require for deletion of personal data from our database via <a href="suporte@teksto.com">Contact Us</a>, however, some information is essential for TEKSTO's service to work properly, for example, email, and removing this kind of data may make it impossible for the user to keep using the TEKSTO app. In case it occurs, the user will be forwarded to the complete deletion of their account from our system.</li>
```
sponsored by Grammarly